src/NServiceBus.Newtonsoft.Json/NewtonsoftSerializer.cs

```diff
@@ -11,9 +11,14 @@
     /// </summary>
     public class NewtonsoftSerializer : SerializationDefinition
     {
+
         /// <summary>
         /// Provides a factory method for building a message serializer.
         /// </summary>
+        [ObsoleteEx(
+        TreatAsErrorFromVersion = "3.0",
+        RemoveInVersion = "4.0",
+        ReplacementTypeOrMember = nameof(NewtonsoftJsonSerializer))]
```

Comment:
I think this obsoletion message should be on the type. The user never calls this method, the framework does.

Reply:
It was meant to be - this was unintentional - shall move it.